man/tpm2_changeauth.1.md

```diff
@@ -19,10 +19,9 @@ authorization values.
 # OPTIONS
 
   * **-o**, **--owner-passwd**=_OWNER\_PASSWORD_:
-    The new owner authorization value.
-
-    Passwords should follow the "password authorization formatting standards",
-    see section "Authorization Formatting".
+    
+    The new owner authorization value.Passwords should follow the "password 
```

Comment:
Missing a space between "value." and "Passwords"

Reply:
@joshuagl thanks.